docs/usage/secrets_rotation.md

```diff
@@ -49,6 +49,18 @@ These client certificates are not revocable and thus a full credential rotation
 
 You can check the `.status.credentials.rotation.kubeconfig` field in the `Shoot` to see when the rotation was last initiated or last completed.
 
+#### Cluster Certificate Authority
+
+*Name*: `<shoot-name>.ca-cluster`
+
+*Description*: Certificate Authority (CA) public key of the Cluster (`Secret` key: `ca.crt`).
+
+This CA is used for signing serving certificates of the Shoot's API server. Hence, the public key contained in this `Secret` can be used to verify the API server's identity when communicating with its public endpoint (e.g. as `certificate-authority-data` in a Kubeconfig).
+This is the same public key that is also contained in the Kubeconfig's `certificate-authority-data` field.
+
+*Rotation*: Not supported yet, but work is in progress. See [gardener/gardener#3292](https://github.com/gardener/gardener/issues/3292) and [GEP-18](https://github.com/gardener/gardener/blob/release-v1.42/docs/proposals/18-shoot-CA-rotation.md) for more details.
+During an ongoing CA rotation operation, this `Secret` contains a certificate bundle of both the old and the new CA certificate.
```

Comment:
```suggestion
During an ongoing CA rotation operation, this `Secret` contains a certificate bundle of both the old and the new CA certificate.
```

I would drop this here (implementation detail). Better end-users are always expecting a bundle (multiple certs).